web/src/settings_invites.ts

```diff
@@ -103,9 +122,14 @@ function populate_invites(invites_data: {invites: Invite[]}): void {
                 item.expiry_date_absolute_time = timerender.absolute_time(item.expiry_date * 1000);
             }
             item.is_admin = current_user.is_admin;
-            item.disable_buttons =
+            item.disable_revoke_button =
                 item.invited_as === settings_config.user_role_values.owner.code &&
                 !current_user.is_owner;
+            item.disable_edit_button =
+                (item.invited_as === settings_config.user_role_values.owner.code &&
+                    !current_user.is_admin) ||
+                !settings_data.user_can_create_multiuse_invite();
+            item.disable_resend_button = item.disable_edit_button;
```

Comment:
This is not correct. Resend button is only used for email invites, so disabling or enabling resend button should not depend on permission to create multiuse invites. If you see the code in main, both revoke and resend buttons were disabled using same variable, hence they should be disabled for same cases.